src/profile-logic/marker-data.js

```diff
@@ -162,6 +163,27 @@ export function getSearchFilteredMarkerIndexes(
   return newMarkers;
 }
 
+export function getTabFilteredMarkerIndexes(
```

Comment:
Please add a code comment explaining this selector, it would be nice for someone who's not familiar with the feature.